src/components/QuestsBanner/QuestsBanner.module.scss

```diff
@@ -53,7 +51,7 @@ $bannerCarouselWidth: 747px;
   flex: 1;
   min-width: 0;
   max-width: 100%;
-  gap: 40px;
+  gap: 24px;
```

Comment:
let's use the spacing css vars for this

Reply:
Ah crap, cheers for noticing, was rushing on that part 🙌